src/Command/SearchSettingsCommand.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Algolia\SearchBundle\Command;
+
+
+use Algolia\SearchBundle\Settings\SettingsManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+abstract class SearchSettingsCommand extends ContainerAwareCommand
+{
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        if ($indexList = (array) $input->getOption('indices')) {
+            $indexList = explode(',', $indexList);
+        }
+
+        $params = [
+            'indices' => $indexList,
+            'extra' => $input->getArgument('extra'),
+        ];
+
+        $projectDir = $this->getContainer()->get('kernel')->getProjectDir();
+        $settingsDir = $projectDir.'/config/settings/algolia_search';
```

Comment:
mh. not sure about this harcoded path there.

What about : 
1. projects that are on Symfony 3.4 and doesn't have this directory structure,which was introduced in SF4.0 ?
2. project doesn't follow standard architecture ?

Reply:
I didn't want to add a new entry to the config but in the end it will be the best option. Pus, I can remove the `$settingsDir` arg from method. Thanks for your feedback! 💯